fixtures/Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+certifi = "==2020.6.20"
+cffi = "==1.14.2"
+chardet = "==3.0.4"
+colorama = "==0.4.3"
+cryptography = "==3.3.2"
+idna = "==2.10"
+iso8601 = "==0.1.12"
+pycparser = "==2.20"
+python-dateutil = "==2.8.1"
+requests = "==2.24.0"
+securesystemslib = "==0.20.1"
+six = "==1.15.0"
+tuf = "==0.14.0"
+urllib3 = "==1.25.10"
+PyNaCl = "==1.4.0"
+flake8 = "*"
+
+[requires]
+python_version = "3.9"
```

Comment:
If we just updating  `generate_fixtures.py` would need a new Pipfile?

While I hope we get all the fixture generation converted to use `FixtureBuilder` quickly if we run into problem and it takes longer I would rather not have the repo be in this confusing state of having 2 steps create fixtures.

Related to that I wonder how hard it would be create a github actions to run  `generate_fixtures.py` and confirm that our fixtures do not change. Because anytime you update the generation of fixtures that change should be committed and we want to enforce that.